dev/com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0</name>
```

Comment:
Again, I think we should leave out "operators"

Reply:
Raised #4492 as a reminder to get EJ's steer.